libraries/ag-charts-test/src/canvas/mock-canvas.ts

```diff
@@ -1,8 +1,19 @@
-import type { Canvas } from 'canvas';
+import { Canvas, loadImage } from 'canvas';
 import { Image, createCanvas } from 'canvas';
 
 import { mockCanvasText } from './mock-canvas-text';
 
+// node-canvas does not support createImageBitmap() yet (https://github.com/Automattic/node-canvas/issues/876).
+// However, the Canvas.drawImage(img,...) method does accept a Canvas-type img parameter. So use as new Canvas
+// as an ImageBitmap.
+(Canvas.prototype as any).transferToImageBitmap = function () {
+    const that: Canvas = this;
+    const { width, height } = that;
+    const bitmap = new Canvas(width, height);
+    bitmap.getContext('2d').drawImage(that, 0, 0, width, height);
+    return bitmap;
+};
+
```

Comment:
Note: This implementation is not perfect. Some of the snapshots have bizarre rendering problems on faded legend items. However, I think that this implement is good enough for the purposes of moving ahead the PR.

Reply:
There was a bug in the sprite rendering code which meant that the sprite canvas wasn't consistently cleared, which may have been the root cause of this - now fixed.